fs/fs.go

```diff
@@ -349,9 +349,10 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 	go func() {
 		rErr := fmt.Errorf("failed to resolve target")
 		for _, s := range src {
-			sociDesc := ocispec.Descriptor{}
-			if desc, ok := c.imageLayerToSociDesc[s.Target.Digest.String()]; ok {
-				sociDesc = desc
+			sociDesc, ok := c.imageLayerToSociDesc[s.Target.Digest.String()]
+			if !ok {
+				rErr = errors.Wrapf(rErr, "failed to resolve layer %q from %q: %v", s.Target.Digest, s.Name, "no ztoc for the layer")
```

Comment:
How about instead of `failed to resolve layer`, we use something like `unable to resolve layer, mounting locally`?